### PR TITLE
fix(android)(7_5_X): amend attributedString diff condition

### PR DESCRIPTION
**Pull request by garymathews**
- Cherry-pick of https://github.com/appcelerator/titanium_mobile/pull/10575

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-20266)

### CODE REVIEW

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java [hunk 1]
```diff
@@ -902,9 +902,11 @@ public void populateViews(KrollDict data, TiBaseListViewItem cellContent, TiList
...
 
 		//process listItem properties
-		KrollDict listItemDiff = cellContent.getViewItem().generateDiffProperties(listItemProperties);
-		if (!listItemDiff.isEmpty()) {
-			listItem.processProperties(listItemDiff);
+		if (cellContent.getViewItem().getView() != null) {
+			KrollDict listItemDiff = cellContent.getViewItem().generateDiffProperties(listItemProperties);
+			if (!listItemDiff.isEmpty()) {
+				listItem.processProperties(listItemDiff);
+			}
```
Comment (jquick-axway) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java]: We need to get rid of the `getView()` part from the `null` check to fix the [TIMOB-26719](https://jira.appcelerator.org/browse/TIMOB-26719) issue.

**Review by jquick-axway — CHANGES_REQUESTED** [review]

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java (hunk 1 above)
Comment (garymathews) [reply android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java]: Yep, cherry-picked into this PR 👍

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Review by ssjsamir — APPROVED** [review]
**FR Passed** When scrolling up and down fast, ListView is now rendered correctly, tested with the test case mentioned in the original PR. 

**Test Environment**
```
Google pixel xl 7.1.1 sim
APPC CLI: 7.0.9
Operating System Name: Mac OS Mojave
Operating System Version: 10.14.2
Node.js Version: 8.9.1
Xcode 10.1
```

**Pull request merged**